structs.go

```diff
@@ -1699,6 +1699,8 @@ type GuildMemberParams struct {
 	Nick string `json:"nick,omitempty"`
 	// Array of role ids the member is assigned
 	Roles *[]string `json:"roles,omitempty"`
+	// Timeout user until as ISO8601 timestamp
+	CommunicationDisabledUntil string `json:"communication_disabled_until,omitempty"`
```

Comment:
We usually use `time.Time` for timestamps.
Additionally this field must be both omittable and nullable (to unmute someone, a `null` value must be set).
There are two ways to implement that:
- Custom `MarshalJSON`
- `NullTime` type

I'm not really sure which one would be the best here. Additionally since there's already a `GuildMemberTimeout`, which serves the same purpose and more fit for the task, I think it would be the best to leave this field out for now.